WISE-Community/WISE-API
# chore(Peer Grouping): Rename Peer Group Activity to Peer Grouping

You may want to test this along with the WISE-Client branch ```issue-552-peer-group-activity-authoring```.

You will need to run these database changes.

```
rename table peer_group_activities to peer_groupings;
alter table peer_groupings rename index peerGroupActivitiesUniqueRunTag to peerGroupingsUniqueRunTag;
alter table peer_groupings drop foreign key peerGroupActivitiesRunIdFK;
alter table peer_groupings drop index peer_group_activities_run_id_index;
alter table peer_groupings add index `peerGroupingsRunIdIndex` (`runId`);
alter table peer_groupings add constraint `peerGroupingsRunIdFK` foreign key (`runId`) references `runs` (`id`);

alter table peer_groups rename column peerGroupActivityId to peerGroupingId;
alter table peer_groups drop foreign key peerGroupActivityIdFK;
alter table peer_groups drop index peerGroupActivityIdFK;
alter table peer_groups add constraint `peerGroupingIdFK` foreign key (`peerGroupingId`) references `peer_groupings` (`id`);
```

## Changes
- Renamed PeerGroupActivity to PeerGroupings in files, folders, functions, and variable names

## Test
- Make sure Peer Chat works in the VLE 
- Make sure Peer Chat works in the Authoring Tool

Closes #114
